README.md

```diff
@@ -20,7 +20,22 @@ setting this up currently requires some effort:
 
 you should now be able to run the code in examples/
 
-# todo
+### For Mac OS
```

Comment:
do you have a reason to use brew and xquartz?
i'm not using xquartz, just plan9port that talks directly to macos. i would think that's a better experience than going through an x server on macos.

(also, i haven't done github PR's from this side yet, might take a while to get the hang of it)

Reply:
Hmm, my build of plan9port doesn't work without `--with-x11` with brew option :/

Reply:
have you tried building plan9port directly? that should be the most straightforward way to get it installed. and i can only imagine it's the only way recommended by plan9port. it should be as easy as cloning and running ./INSTALL.